New page added for emoji skin tones (#4133)
mattermost/docs: source/help/messaging/emoji-skin-tones.rst

Emoji Skin Tones
================

Emoji are pictures that can communicate emotions, gestures, intent, and so much more. The Mattermost Emoji Picker uses neutral skin tones by default. You can customize the skin tone for many emoji using a native emoji picker.

> Selecting Emoji

Reviewer: ```suggestion
Selecting Emojis
```

---------------

> Click the smile icon inside the message input box to open the Mattermost Emoji Picker.

Reviewer: ```suggestion
Select the smile icon inside the Mattermost message input box to open the Emoji Picker.
```

.. image:: ../../images/selectemoji.png
  :alt: Open the Mattermost Emoji Picker.

Selecting Emoji Skin Tones
--------------------------

> To access alternate skin tones, use the native emoji pickers available on your platform or device.

Reviewer: ```suggestion
To access alternate skin tones, open Mattermost on your device and use the native emoji pickers provided on your platform or device.
```

Reviewer: For the steps below, does the user need to be in Mattermost first? Not sure if we need to state that for all the sections or whether my suggestion will suffice? 

Also, not sure if it's something we should note but this only applies to "hand" emojis (not smileys etc) and the selection will remain as such going forward until/unless changed.

**Windows 10**

> 1. Use the keyboard shortcut: Windows Key + . (period) to open the native emoji picker for Windows 10.

Reviewer: ```suggestion
1. Use the keyboard shortcut: Windows Key+. (period) to open the native emoji picker for Windows 10.
```

2. Select an alternate skin tone and then select an emoji.

.. image:: ../../images/windows10emojiskintones.png
  :alt: Select alternate emoji skin tones on Windows 10 systems.
  :scale: 70

> **MacOS**

Reviewer: ```suggestion
**macOS**
```

> 1. Use the keyboard shortcut CMD+CTRL+Space Bar to open the native emoji picker for MacOS.

Reviewer: ```suggestion
1. Use the keyboard shortcut CMD+CTRL+Space Bar to open the native emoji picker for macOS.
```

> 2. Click and hold on an emoji to select an alternate skin tone.

Reviewer: ```suggestion
2. Select and hold on an emoji to select an alternate skin tone.
```

.. image:: ../../images/macosemojiskintones.png
  :alt: Select alternate emoji skin tones on MacOS systems.
  :scale: 80

**iPhone and iPad**

> 1. Open the iPhone Emoji Keyboard.

Reviewer: ```suggestion
1. Open the emoji keyboard.
```

2. Touch and hold an emoji to select an alternate skin tone.

.. image:: ../../images/mobile/appleemojiskintones.png
  :alt: Select alternate emoji skin tones on Apple devices.
  :scale: 30

.. tip::
  If you don't see the emoji keyboard for your Apple device, you can enable it. See `Use emoji on your iPhone, iPad, and iPod touch <https://support.apple.com/en-us/HT202332>`__ for details.

**Android**

> 1. Open an Android Emoji Keyboard.

Reviewer: ```suggestion
1. Open an Android emoji keyboard.
```
